cluster-operator/src/main/resources/entityTopicOperatorDefaultLoggingProperties

```diff
@@ -4,6 +4,10 @@ appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,nnnnn} %-5p [%t] %c{1}:%L - %m%n
+appender.console.filter.filter1.type = MarkerFilter
+appender.console.filter.filter1.onMatch = ACCEPT
+appender.console.filter.filter1.onMismatch = NEUTRAL
+appender.console.filter.filter1.marker = KafkaTopic(namespace/name)
```

Comment:
What do these settings do?

Reply:
we are creating a filter named `filter1`. It is a Marker filter accepting log entries that do have a marker equal to `KafkaTopic(namespace/name)`. If the marker does not equal, `NEUTRAL` action is performed. By default, it is `ACCEPT`. So this configuration does nothing. It is just an example, where you can easily change `NEUTRAL` to `DENY` and adjust `marker` to something like `KafkaTopic(myproject/my-topic)` to see related topic messages in logs only.

Reply:
Why are we putting this example in the default logging properties for an operator?

We should document the configuration in the documentation.

Reply:
Maybe a stupid note because I do not know how it works and what is the matching based on. But today, the reconciliation part looks like this:
```
Reconciliation #36(timer) KafkaConnect(myproject/my-connect)
```

So it contains the sequence number and trigger `#36(timer)`. I understand this is irrelevant for the Log4j configuration ... but it is really super important for analyzing the logs. So please make sure this is still in the actual log message.

Reply:
The marker is actually some kind of annotation or label metadata. It is not visible in the logs. It servers as the key to filter.

Reply:
That is fine ... just make sure the information about the trigger is still in the log messages. ;-)